src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStartTask.java

```diff
@@ -43,13 +45,22 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
   @TaskAction
   public void startAction() throws AppEngineException, IOException {
 
+    // Add a listener to write to a file for non-blocking starts, this really only works
+    // when the gradle daemon is running (which is default for newer versions of gradle)
+    File logFile = File.createTempFile("dev_appserver", ".out", getTemporaryDir());
```

Comment:
Is it intentional that this log file could be erased after the execution of the task? The docs for `getTemporaryDir()` mention: "There are no guarantees that the contents of this directory will be kept beyond the execution of the task."

I would imagine many people may execute the task, watch it fail, and check the log to see why it failed but this file might be erased.